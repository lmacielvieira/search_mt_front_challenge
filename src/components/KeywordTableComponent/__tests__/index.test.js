import React from 'react'
import {shallow} from 'enzyme'
import {KeywordTableComponent} from '../index'

describe('<KeywordTableComponent />', () => {
  it('renders <KeywordTableComponent />', () => {
    const wrapper = shallow(<KeywordTableComponent />)
    expect(wrapper).toMatchSnapshot()
  })
})
