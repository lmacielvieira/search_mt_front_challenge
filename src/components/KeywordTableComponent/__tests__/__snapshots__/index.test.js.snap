// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<KeywordTableComponent /> renders <KeywordTableComponent /> 1`] = `ShallowWrapper {}`;
